### PR TITLE
Remove hardcoded network value

### LINKED ISSUE #129: Hardcoded network name
The `NetworkName` value is hardcoded and required to be changed for example run against localnet.

https://github.com/Zondax/rosetta-filecoin-proxy/blob/40df4bab6de053b504a20e7ac9edc7e22b8cb27e/rosetta/services/constants.go#L19

### PR DESCRIPTION
**Pull request by rllola**
Detect network value at runtime using environment variable or default to 'mainnet'.

closes #129

### CODE REVIEW

File: rosetta/services/constants.go [hunk 1]
```diff
@@ -1,6 +1,20 @@
 package services
 
+import "os"
+
+func getEnvNetwork() string {
+	if value, ok := os.LookupEnv("NETWORK"); ok {
+		if value == "mainnet" || value == "testnet" {
+			return value
```
Comment (raynaudoe) [inline rosetta/services/constants.go:8]: Maybe adding here a log output saying "unrecognized network <value> . Defaulting to mainnet" or something like that ?

**Review by raynaudoe — APPROVED** [review]

File: rosetta/services/constants.go (hunk 1 above)
Comment (rllola) [reply rosetta/services/constants.go:11]: Made the modification. Should be better now.

**Comment by rllola** [comment]
I am not sure we need this. As far as I can tell the Network name is taken directly from the Lotus node ? The only time when we use this variable it is for tests. Maybe we should simply remove this variable and introduce it only in tests file. It would avoid confusion.

**Comment by jleni** [comment]
Any news here? It would be nice to close this

**Comment by rllola** [comment]
I don't understand why the check won't pass.

**Review by raynaudoe — APPROVED** [review]

**Pull request merged**